playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -482,7 +482,7 @@
     - install:oauth-credentials
 
 - name: create service worker users
-  shell: "{{ edxapp_venv_bin }}/python ./manage.py lms --settings={{ edxapp_settings }} --service-variant lms manage_user {{ item.username}} {{ item.email }} --unusable-password {% if item.is_staff %} --staff{% endif %}"
+  shell: ". {{ edxapp_app_dir }}/edxapp_env && {{ edxapp_venv_bin }}/python ./manage.py lms --settings={{ edxapp_settings }} --service-variant lms manage_user {{ item.username}} {{ item.email }} --unusable-password {% if item.is_staff %} --staff{% endif %}"
```

Comment:
All of the changes in this file are fine because `edxapp_app_dir` is defined in this `edxapp` role.